ajax/libs/react-markdown-editor-lite/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "react-markdown-editor-lite",
+  "description": "A light-weight Markdown editor of React component",
+  "keywords": [
+    "markdown",
+    "html",
+    "editor",
+    "parser",
+    "react",
+    "component",
+    "plugins",
+    "pluggable"
+  ],
+  "author": "HarryChen && ShuangYa",
+  "license": "MIT",
+  "homepage": "https://github.com/HarryChen0506/react-markdown-editor-lite#readme",
```

Comment:
I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Reply:
> I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Yes, but in face, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content

Reply:
Yes, but in fact, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content. So I think it ’s better to use GitHub as its homepage

Reply:
> I think this is the home page https://harrychen0506.github.io/react-markdown-editor-lite/... Am I wrong?

Yes, but in fact, this website is not maintained frequently, and the content is relatively old. Only GitHub has complete content. So I think it ’s better to use GitHub as its homepage

Reply:
OK. LGTM